utam-core/src/main/java/utam/core/framework/base/BaseRootPageObject.java

```diff
@@ -17,7 +17,7 @@
 
   @Override
   public Object load() {
-    log("wait for a root element to be found");
+    log("load the object - wait for a root element to be found");
```

Comment:
Nitpick: the object feels very generic to me, what object are we talking about?

Reply:
logger has a prefix that explicitly states what its applied to

Reply:
That's good then resolving this